db/migrate/20150714144500_review_discovery_permissions.rb

```diff
@@ -0,0 +1,16 @@
+class ReviewDiscoveryPermissions < ActiveRecord::Migration
+  def up
+    if mgr = Role.find_by_name("Discovery Manager")
+      perms = []
+      perms << "submit_discovered_hosts" if Permission.find_by_name("edit_discovered_hosts")
+      perms << "auto_provision_discovered_hosts" if Permission.find_by_name("provision_discovered_hosts")
+      perms << "create_discovery_rules" if Permission.find_by_name("new_discovery_rules")
+      mgr.add_permissions!(perms)
+    end
+    Permission.find_by_name("new_discovery_rules").try(:destroy)
+  end
+
+  def down
+    # Foreman plugin initializer will re-create
```

Comment:
Yeah bad comment, the fact is - we don't need to rollback anything in this case. Changed the comment.